src/components/__tests__/Components.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from '../../pages/Home';
+import Quotes from '../../pages/Quotes';
+import Calculator from '../Calculator';
+
+describe('Calculater component test', () => {
+  test('builds the snapshot of the Calculator component', () => {
+    const tree = renderer.create(<Calculator />);
+    expect(tree.toJSON()).toMatchSnapshot();
+  });
+
+  test('builds the snapshot of the Home component', () => {
+    const tree = renderer.create(<Home />);
+    expect(tree.toJSON()).toMatchSnapshot();
+  });
+
+  test('builds the snapshot of the Quotes component', () => {
+    const tree = renderer.create(<Quotes />);
+    expect(tree.toJSON()).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Kindly add some integration tests to your application where you test user interaction and check out the results. For example you can check calculator displaying the right thing by using test id's on the display and then clicking buttons using react testing library. Below is just an arbitrary example
``` javascript
   render(<Calculator />);
    const btnElement = screen.getByText(/6/i);
    const resultElement = screen.getByTestId('result-id');
    fireEvent.click(btnElement);
    expect(resultElement).toHaveTextContent('6');
```